### PR TITLE
feat(git-practice): Ariel's rehearsal

### PR DESCRIPTION
**Pull request by arielarmijo**
Practice workshop

### CODE REVIEW

**Review by RickyCode — COMMENTED** [review]
please add a blank line at the end of the file

**Comment by arielarmijo** [comment]
> 
of course.

File: ariel-git-practice/test-file.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Archivo de prueba
+
+- item 1
+- item 2
+- item 3
```
Comment (RickyCode) [inline ariel-git-practice/test-file.md]: add blank line

Comment (mframirezfuentes) [inline ariel-git-practice/test-file.md:5]: create a new item, please

File: ariel-git-practice/test-file.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Archivo de prueba
```
Comment (RickyCode) [inline ariel-git-practice/test-file.md:1]: nice title

**Review by carrizotamara — COMMENTED** [review]
nice job

File: ariel-git-practice/test-file.md (hunk 1 above)
Comment (arielarmijo) [reply ariel-git-practice/test-file.md:5]: sure, taked it for granted!

**Comment by arielarmijo** [comment]
> nice job

thanks!

File: ariel-git-practice/test-file.md (hunk 1 above)
Comment (arielarmijo) [reply ariel-git-practice/test-file.md]: done

**Review by RickyCode — APPROVED** [review]

**Review by mframirezfuentes — APPROVED** [review]

**Review by carrizotamara — APPROVED** [review]

**Pull request merged**